src/lxc/cmd/lxc_user_nic.c

```diff
@@ -457,9 +457,6 @@ static char *find_line(char *buf_start, char *buf_end, char *name,
 		if (!end_of_word)
 			return NULL;
 
-		if (strncmp(buf_start, net_type, strlen(net_type)))
-			*found = false;
```

Comment:
If I see this right, `net_type` is now entirely unused, which doesn't feel right. I suppose a mismatching type could set `found_dev` to `false`?

Reply:
It really doesn't feel right, but is no change in behavior since before `*found` was set to `false` which is the same as its initial value and there is no setting to `true` before this line.

Reply:
Since I didn't dive into the `cull_entries` function, I wasn't sure about changing the old behavior. But `find_line` probably really should return only one `found` value if all parameters (`!= NULL`) are matching.

Reply:
Did you ever get around to updating the patch to do that?

Reply:
Sorry, I didn't have any time to do this.
Will try to look into it next weekend, but can't promise anything.